readme.md

```diff
@@ -45,6 +45,10 @@ Type: `number`
 
 The maximum number of items before evicting the least recently used items.
 
+#### maxAge
+
+If maxAge milliseconds is specified for the cache, the items will have an expiration time defined by this maxAge. Lazy expiration upon next get, peek call of the item.
```

Comment:
I think this could be written more succinctly. You don't have to mention `maxAge` as that's clear from the context. You're also missing a type annotation.